ee/clickhouse/queries/funnels/funnel_trends_persons.py

```diff
@@ -8,6 +8,13 @@
 
 
 class ClickhouseFunnelTrendsPersons(ClickhouseFunnelTrends):
+    def run(self, *args, **kwargs):
+        if len(self._filter.entities) == 0:
+            return []
+
+        results = self._exec_query()
+        return self._format_results(results), len(results) > 99
```

Comment:
Should we store this `99` and the relevant `100` limit in a shared constant or in a param for the queries? Seems too easy to get these accidentally out of sync.

A param would also make sense since a nice paginated infinite list loader is one which gradually increases the nr of rows read to limit queries. E.g. load 100 items initially, then "load more" adds 200, then 400, then 800, etc.

Eventually our infinitely loading pages should 

A nice infinite scroll is one where we first fetch 100 rows, then 200, then 400, then 800